docker-compose.yml

```diff
@@ -154,7 +154,7 @@ services:
       REDIS_PASSWORD: password
       CODETABLE_REFRESH_ENABLED: ${CODETABLE_REFRESH_ENABLED:-true}
       CODETABLE_REFRESH_CRON: ${CODETABLE_REFRESH_CRON:-0 0/15 * * * *}
-      H2_DATASOURCE: ${H2_DATASOURCE:-jdbc:h2:file:./data/h2-v2}
+      H2_DATASOURCE: ${H2_DATASOURCE:-jdbc:h2:mem:testdb}
```

Comment:
Since there is a model change, the H2 database needs to be dropped and recreated.  Easiest is to just make the db in-memory.